Gem/Code/Source/Components/CharacterComponent.h

```diff
@@ -19,6 +20,7 @@ namespace MultiplayerSample
 {
     class CharacterComponent
```

Comment:
Just a heads up that I've renamed this component to NetworkCharacterComponent upon the advice of the UX team. AnimatedHitVolumes is also now NetworkHitVolumes, so aside from the camera and WASD movement all of our multiplayer sample components are prefixed with 'Network'. It should merge cleanly but it's good to let you know this is coming.